recipe/meta.yaml

```
@@ -39,6 +45,19 @@ requirements:
     - pip
     - pytorch
     - pytorch =*={{ torch_proc_type }}*
+    {% if cuda_major >= 12 %}
+    - libcufft-dev  # [linux or win]
+    - libcurand-dev  # [linux or win]
+    - libcublas-dev  # [linux or win]
+    - cuda-nvrtc-dev  # [linux or win]
+    - cuda-nvtx-dev  # [linux or win]
+    - cuda-profiler-api  # [linux or win]
+    - cuda-nvrtc-dev  # [linux or win]
+    - libcusparse-dev  # [linux or win]
+    - libcusolver-dev  # [linux or win]
+    # No cuda-driver-dev in windows
```

It looks like this package links to `cudart` dynamically so needs it at runtime. Inferring this from [this CI warning]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=837161&view=logs&j=b7a895b1-8909-5023-4a22-40a4698fe576&t=821caef9-b8fa-5ab7-47a2-7e7cdae7737e&l=822 ):

```
WARNING (pyg-lib,lib/python3.10/site-packages/libpyg.so): Needed DSO lib/libcudart.so.12 found in ['cuda-cudart-dev', 'cuda-cudart-dev_linux-64', 'cuda-cudart_linux-64', 'cuda-cudart_linux-64', 'cuda-cudart', 'cuda-cudart']
WARNING (pyg-lib,lib/python3.10/site-packages/libpyg.so): .. but ['cuda-cudart-dev', 'cuda-cudart-dev_linux-64', 'cuda-cudart_linux-64', 'cuda-cudart_linux-64', 'cuda-cudart', 'cuda-cudart'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```

So would suggest including it as well

```suggestion
    - cuda-cudart-dev
    # No cuda-driver-dev in windows
```